packages/components/src/AudioPlayer/styles.css.ts

```diff
@@ -52,7 +52,7 @@ export default {
   ]),
   buttonBar: style({
     position: 'relative',
-    justifyContent: 'center',
+    justifyContent: 'center !important',
```

Comment:
Salt styles were taking precedence, even when setting this in the vanilla extract theme.
!important flag used to make sure it is always applied